doc/nrf/getting_started/recommended_versions.rst

```diff
@@ -176,6 +176,42 @@ Other versions might also work, but are not verified.
          * - West
            - |west_recommended_ver_darwin|
 
+.. _gs_git_auth:
+
+Git authentication
+******************
+
+During the process of :ref:`getting the |NCS| code <cloning_the_repositories>`,
+in some cases running ``west update`` might require :ref:`west <zephyr:west>` to
+fetch from private repositories on GitHub. Because the `west manifest file`_
+uses ``https://`` URLs instead of ``ssh://``, you may be prompted to type your
+GitHub username and Personal Access Token multiple times; GitHub has a
+comprehensive `documentation page
+<https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-authentication-to-github>`_
+on the subject.
+In many cases (including Windows) the Git installation includes `Git Credential Manager
+<https://github.com/git-ecosystem/git-credential-manager>`_, which will handle
+GitHub authentication.
```

Comment:
```suggestion
When :ref:`getting the |NCS| code <cloning_the_repositories>`, sometimes running ``west update`` requires :ref:`west <zephyr:west>` to fetch from private repositories on GitHub.

Because the `west manifest file`_ uses ``https://`` URLs instead of ``ssh://``, you may be prompted to type your GitHub username and Personal Access Token multiple times.
   GitHub has a comprehensive `documentation page <https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-authentication-to-github>`_ on the subject.
   In many cases (including Windows), the Git installation includes `Git Credential Manager <https://github.com/git-ecosystem/git-credential-manager>`_, which will handle GitHub authentication and is the recommended method for handling GitHub authentication.
```

Reply:
Please move the links to the links.txt